### ISSUE TITLE
CMS-Export Cleanup (part 2)

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#4462]
Follow up to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4429:

Tasks: 
- [x] Remove documentation (leave commits here for historical reference)
- [x] Remove cms-asset endpoint (won't work with EFS)
- [x] Remove existing files/directories on PROD, e.g. docroot/sites/default/files/cms-export|asset folders
- [x] Do human thinking and figure any other cleanup tasks and do them
- [x] Remove "Run CMS Content Export playbook" stage in ansible/Jenkinsfiles/cms/cms-deploy-live.groovy https://github.com/department-of-veterans-affairs/devops/pull/8675

**Comment by indytechcook** [comment]
I removed the modules and the readmes in #4262 

The `README/deploy-process.md`

**Comment by indytechcook** [comment]
`Remove cms-asset endpoint (won't work with EFS)` 
 
is a PHP sone and I'll do that in https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4535

**Comment by olivereri** [comment]
`READMES/cms-export.md` needs to be removed as well, correct?

**Comment by indytechcook** [comment]
I removed that here: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4535

**Comment by olivereri** [comment]
Just to be safe docroot/sites/default/files _is_ the assets folder that _won't go away_ until EFS is ready. Inside that folder are 2 sub folders called cms-export-content and cms-export-files which should be deleted. I don't see anything else in there export related.

@indytechcook Do I need to wait for 4535 to be merged and deployed before cleaning up the assets folder?

**Comment by indytechcook** [comment]
No, the jobs are already removed from the deploy so the files won't be generated anymore.

**Comment by ElijahLynn** [comment]
> Inside that folder are 2 sub folders called cms-export-content and cms-export-files which should be deleted.

correct

**Comment by olivereri** [comment]
Searching through the va.gov-cms repo for the below search terms. Most of these are js files. Anything else is caught up #4535

.*cms.*asset.*
.*cms.*export.*
.*export.*cms.*
.*export.*asset.*
.*asset.*export.*

docroot/vendor/va-gov/web/src/site/stages/build/options.js 19,69
docroot/vendor/va-gov/web/script/cms/compare.sh 3,4,6
docroot/vendor/va-gov/web/script/cms/config.js 9,14,16
docroot/vendor/va-gov/web/script/cms/test-bundle-transformers.js 11,14,73,83
docroot/vendor/va-gov/web/script/content-build-help.js 7,60
docroot/vendor/va-gov/web/src/site/stages/build/drupal/api.js 48,150
docroot/vendor/va-gov/web/src/site/stages/build/drupal/metalsmith-drupal.js 32,33,34,194
scripts/content/VACMS-3450-testing.sh 8,12,14
script/cms/compare.sh 6
script/cms/config.js 14,16
script/cms/test-bundle-transformers.js 11,14,73,83
script/content-build-help.js 7,60
src/site/stages/build/drupal/api.js 48,150
src/site/stages/build/drupal/metalsmith-drupal.js 32,33,34,194
src/site/stages/build/options.js 19,69
docroot/vendor/va-gov/web/script/cms/genBundleSchema.js 28
docroot/vendor/va-gov/web/script/remove-unnecessary-raw-entity-files.sh 5,11,33
docroot/vendor/va-gov/web/src/site/stages/build/process-cms-exports/tests/transformer-helpers.unit.spec.js 14
docroot/vendor/va-gov/web/src/site/stages/build/process-cms-exports/transformers/file.js 4
script/cms/genBundleSchema.js 28

Not sure if this one counts, maybe a different export?
docroot/modules/custom/va_gov_build_trigger/src/CommandExportable.php 6 

Just a comment.
docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.drush.inc 5 

## On the Devops repository there are still a couple hits:

Deploy live job sill calls the stage "Run CMS Content Export playbook"
ansible/Jenkinsfiles/cms/cms-deploy-live.groovy
ansible/Jenkinsfiles/cms-test/cms-deploy-live.groovy

**Comment by indytechcook** [comment]
Reopening and assigning to me to finish this work in the next sprint

**Comment by indytechcook** [comment]
The work didn't get complete because it was blocked by the https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4262. 

The PR in progress is https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4650.

**Comment by ElijahLynn** [comment]
> Reopening and assigning to me to finish this work in the next sprint

Can we split the remaining work out and resize the remaining effort?
